src/libs/ascent/runtimes/ascent_main_runtime.cpp

```diff
@@ -1638,7 +1638,7 @@ AscentRuntime::CreateScenes(const conduit::Node &scenes)
 
     std::string renders_name = names[i] + "_renders";
 
-    if(!flow::Workspace::supports_filter_type("default_render"))
+    if(!flow::Workspace::supports_filter_type("create_renders"))
```

Comment:
`default_render` wasn't a great name, `create_renders` conveys the purpose better